### PR TITLE
Casmuser 3130

### PR DESCRIPTION
**Pull request by keopp-hpe**
## Summary and Scope

Document update to include a "Known Issues" section in the guides.

First "Known Issue" is the workaround for the kdump configuration.

## Issues and Related PRs

* Resolves [CASMUSER-3130](https://jira-pro.its.hpecorp.net:8443/browse/CASMUSER-3130)

## Risks and Mitigations

Low risk

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [X] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: docs/portal/developer-portal/index.md [hunk 1]
```diff
@@ -5,45 +5,58 @@ and operational procedures for Cray EX User Access Nodes (UAN).
 
 ## Table of Contents
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/index.md:6]: this is the first time I remember seeing this file. what do we use it for?

**Review by darrenn-jackson — CHANGES_REQUESTED** [review]
Just a few minor formatting changes, plus a question.

I checked out this branch and successfully built the HTML5 bundles for both guides and published to ITG. The new "Known Issues" section displays just as expected.

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
+# KDUMP Configuration Workaround
+
+There exists a known bug in the `kdump` Ansible role which prevents kdump from working on UAN nodes.  A workaround for this bug exists and is describe below.
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/known_issues/KDUMP_Configuration.md]: Updated.

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+## Workaround
+
+The workaround is to change `kdump_target_uan: yes` to `kdump_target_uan: "True"` in the UAN site.yaml file, then reconfigure the UAN image with the updated CFS configuration.  The following procedure describes how to perform the workaround.
+
+### Procedure
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/known_issues/KDUMP_Configuration.md]: Updated.

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+1. Checkout the `integration` branch or whatever branch is currently used to configure the UANs.
+
+    ```bash
+    ncn-m001# git checkout integration
+    ```
+
+1. Edit the `site.yaml` file and change the lines as shown below:
+
+    **FROM:**
+    ```bash
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/known_issues/KDUMP_Configuration.md]: Updated.

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+        - role: kdump
+          kdump_target_uan: yes
+    ```
+
+    **TO:**
+    ```bash
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/known_issues/KDUMP_Configuration.md]: Updated.

File: docs/portal/developer-portal/uan_admin_guide.ditamap [hunk 6]
```diff
@@ -35,6 +35,9 @@
         <topicref href="troubleshooting/Troubleshoot_UAN_Disk_Configuration_Issues.md"
             format="markdown"/>
     </topicref>
+    <topicref href="Known_Issues.md" format="markdown">
+        <topicref href="known_issues/KDUMP_Configuration.md" format="markdown"/>
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/uan_admin_guide.ditamap:39]: Agreed.

File: docs/portal/developer-portal/index.md (hunk 1 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/index.md:6]: This is part of the hugo docs.

**Review by darrenn-jackson — APPROVED** [review]

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+    ```
+
+    1. Monitor the CFS session.  It is named `uan-kdump` in the command line above. Repeat this command until the [status.session] status is "complete" and succeeded is "true".
+
+    ```bash
+    ncn-m001# cray cfs sessions describe uan-kdump
+    ```
+
+    ```bash
+    ncn-m001# cray cfs sessions describe nersc-uan
```
Comment (alanm-hpe) [inline docs/portal/developer-portal/known_issues/KDUMP_Configuration.md:189]: I don't think we should have `nersc-uan` here

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+
+    1. Update the BOS session template with the JSON file.
+
+    ```bash
+    ncn-m001# cray bos sessiontemplates create --file uan-bos-PRODUCT_VERSION.json uan-bos-PRODUCT_VERSION.json
+    ```
+    
+1. Reboot the UAN with the Boot Orchestration Service \(BOS\).
+
+    The new interfaces will be available when the UAN is rebooted. Replace the `UAN_SESSION_TEMPLATE` value with the BOS session template name for the UANs.
```
Comment (alanm-hpe) [inline docs/portal/developer-portal/known_issues/KDUMP_Configuration.md:258]: Should this say "interfaces"?

File: docs/portal/developer-portal/known_issues/KDUMP_Configuration.md (hunk 7 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/known_issues/KDUMP_Configuration.md]: Oh, good catch!  Thanks!

File: docs/portal/developer-portal/Known_Issues.md [hunk 9]
```diff
@@ -0,0 +1 @@
+# Known Issues
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/Known_Issues.md:1]: It matches others that provide just a header in the ditaweb sections.  Not used by hugo.  Just the ditaweb docs.

**Review by alanm-hpe — APPROVED** [review]

**Review by darrenn-jackson — APPROVED** [review]

**Pull request merged**